packages/react-sdk/src/css/normalize.ts

```diff
@@ -37,13 +37,21 @@ const boxSizing = {
  *   box-sizing: border-box;
   }
 */
-const baseStyle = [boxSizing, ...borders, ...outline] satisfies Styles;
+const baseStyle = [
+  boxSizing,
+  ...presets.borders,
+  ...presets.outline,
+] satisfies Styles;
 
 export const div = baseStyle;
 export const address = baseStyle;
 export const article = baseStyle;
 export const aside = baseStyle;
-export const figure = baseStyle;
+export const blockquote = [
+  ...baseStyle,
+  ...presets.blockquote,
+] satisfies Styles;
+export const figure = [...baseStyle, ...presets.margins] satisfies Styles;
```

Comment:
might not need margins, but only vertical margins, need to check browser defaults there